config.yaml

```diff
@@ -293,34 +293,32 @@ commands:
     - must have ecs:container
   task-list:
     executable: "/home/bundle/cog-command"
-    description: Lists registered tasks.
+    description: Lists task definition families and revisions.
+    long_description: Lists task definition families and revisions. Pass a task definition
```

Comment:
That's a good point. I didn't know if we would ever just show the long description or not. If we aren't then I'll stop included the short description.